src/out.jl

```diff
@@ -482,7 +456,56 @@ function write_cum_maps(cum, cellmap::Matrix{T}, cfg, hbmeta, write_max, write_c
             max_curr .= max.(cum.max_curr[i], max_curr)
         end
         postprocess_cum_curmap!(max_curr)
-        write_aagrid(max_curr, "", cfg, hbmeta, max = true)
+        write_grid(max_curr, "", cfg, hbmeta, max = true)
+    end
+
+end
+
+# Write a single band raster, either in .tif or .asc format,
+# inspired by GeoArrays.write()
+function write_raster(fn_prefix::AbstractString,
+                      array,
+                      wkt::AbstractString,
```

Comment:
This too

Reply:
Adding a type for `array` would be ideal here, but I was having trouble (:facepalm:) figuring out how to make it accept any numeric array -- kept getting errors about no matching method no matter what I tried.